layouts/partials/menu-bundle.html

```diff
@@ -23,8 +23,9 @@
             {{ $isAncestor := $this.IsAncestor $current }}
             {{ $id := substr (sha1 $this.Permalink) 0 8 }}
             {{ $hasCollapse := and $this.Params.GeekdocCollapseSection (isset . "sub") }}
+            {{ $doCollapse := or $hasCollapse (default false .Site.Params.GeekdocCollapseAllSections) }}
```

Comment:
I would put it into a single statement:

```suggestion
            {{ $doCollapse := and (isset . "sub") (or $this.Params.GeekdocCollapseSection (default false .Site.Params.GeekdocCollapseAllSections)) }}
```